arraylist/src/main/java/ua/hillel/Main.java

```diff
@@ -0,0 +1,28 @@
+package ua.hillel;
+
+public class Main {
+    public static void main(String[] args) {
+        ArrayList<Integer> arrayList = new ArrayList<>();
```

Comment:
You should always use collections like this: `List<Integer> arrayList = new ArrayList<>();`
An interface is always declared on the left, and the implementation is declared on the right. This is a rule of thumb.